packages/CopyInput/src/CopyInput.js

```diff
@@ -46,48 +47,65 @@ function CopyInput(props) {
 
   return (
     <sc.CopyInput data-pka-anchor="copy-input" {...moreProps}>
-      <Input ref={inputRef} defaultValue={defaultValue} {...extendedInputProps} />
-      <div ref={buttonRef}>
+      <Input ref={inputRef} defaultValue={value} isReadOnly={isReadOnly} {...extendedInputProps} />
+      <div ref={buttonRef} data-pka-anchor="copy-input.button">
         <Button.Icon
           kind="primary"
           onClick={handleButtonClick}
-          onMouseEnter={() => setIsHoverTooltipOpen(true)}
-          onMouseLeave={() => setIsHoverTooltipOpen(false)}
+          onMouseOver={() => setIsHoverTooltipOpen(true)}
+          onMouseOut={() => setIsHoverTooltipOpen(false)}
+          onFocus={() => setIsHoverTooltipOpen(true)}
+          onBlur={() => setIsHoverTooltipOpen(false)}
         >
           <CopyIcon />
         </Button.Icon>
       </div>
       <Popover
+        a11yText={isClickedTooltipOpen ? I18n.t("copyInput.clicked_tooltip") : I18n.t("copyInput.hover_tooltip")}
```

Comment:
This needs to go on the `<Button.Icon>` actually.

Reply:
But it feels weird Button's a11yText would change to "Copied" after you click it

Reply:
Oh, good point! And we have the toast already to announce copied. 
It should just be the hover tooltip instead! 🤦